README.md

```diff
@@ -1,18 +1,18 @@
 # RoboRacing Software [![CircleCI](https://circleci.com/gh/RoboJackets/roboracing-software.svg?style=svg)](https://circleci.com/gh/RoboJackets/roboracing-software)
 
-<img src="https://raw.githubusercontent.com/wiki/RoboJackets/roboracing-software/images/buzz_bigoli.jpg" style="max-height=400px;">
+<img src="https://raw.githubusercontent.com/wiki/RoboJackets/roboracing-software/images/sedanii.png" width="1500" height="500">
```

Comment:
The image looks very low res on my screen. If github will let you, use a larger image and correct aspect ratio (the original 3650x2008).

Also, am I allowed to request small photo credits under this? I am very proud of it.

Reply:
Of course, I credited you in [alt text]